pkg/cache/multi_namespace_cache.go

```diff
@@ -279,10 +279,7 @@ func (c *multiNamespaceCache) List(ctx context.Context, list client.ObjectList,
 		return err
 	}
 
-	allItems, err := apimeta.ExtractList(list)
-	if err != nil {
-		return err
-	}
+	allItems := []runtime.Object{}
```

Comment:
While I agree this change makes sense, its a breaking one - Can we mark it as such?

/ok-to-test

Reply:
Do you have any suggestions?

Reply:
Sorry for the delay. I changed the PR title that's enough